x-pack/plugins/security_solution/public/common/components/visualization_actions/use_lens_attributes.tsx

```diff
@@ -108,10 +108,10 @@ export const useLensAttributes = ({
         ...(applyGlobalQueriesAndFilters ? { query } : {}),
         filters: [
           ...attrs.state.filters,
-          ...(applyGlobalQueriesAndFilters ? filters : []),
           ...pageFilters,
           ...tabsFilters,
           ...indexFilters,
+          ...(applyGlobalQueriesAndFilters ? filters : []),
```

Comment:
if this change was undone, I think all of the snapshot test updates wouldn't be needed

Reply:
Yes, but I think it makes sense to have global query and filters at the bottom, so it takes the highest priority.